app/javascript/__tests__/data/RailsListingPricingTableUnits/listing-pricing-table-units-default.ts

```diff
@@ -0,0 +1,200 @@
+import type { RailsListingPricingTableUnit } from "../../../api/types/rails/listings/RailsListingPricingTableUnit"
```

Comment:
The file that this import is using doesn't exist. It's strange that our linter is not catching this. I'm kind of worried now that there may be other things our linter isn't catching.

Reply:
Good catch, and agreed on the linter. I will look into the linter to make sure that it is correctly flagging these types of things.

I wonder if this file was brought in during Mark's work on the pricing table and was forgotten... It doesn't look like it is being used anywhere in the code base (which might be why the linter didn't catch it?). I am going to delete the file unless there is any objection from @emilyjablonski

Reply:
No objection!